### DISCUSSION TITLE
Issue when downloading ogbn-papers100M

### DISCUSSION THREAD

**Discussion by jschweiz** [snap-stanford/ogb#247] [MAG240M-LSC]
Hi all!
I'm trying to download the ogbn-papers100M graph with the Library-Agnostic Loader. The file is 50Gb, and it downloads a file named papers100M-bin.zip. The download seems to work fine, but when it tries to extract the zip file, it fails with an error:

`>>> dataset = NodePropPredDataset(name='ogbn-papers100M')
This will download 56.17GB. Will you proceed? (y/N)
y
Downloading http://snap.stanford.edu/ogb/data/nodeproppred/papers100M-bin.zip
Downloaded 50.00 GB: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 57519/57519 [36:56<00:00, 25.95it/s]
Extracting dataset/papers100M-bin.zip
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "~/python3.6/site-packages/ogb/nodeproppred/dataset.py", line 63, in __init__
    self.pre_process()
  File "~/python3.6/site-packages/ogb/nodeproppred/dataset.py", line 90, in pre_process
    extract_zip(path, self.original_root)
  File "~/python3.6/site-packages/ogb/utils/url.py", line 87, in extract_zip
    with zipfile.ZipFile(path, 'r') as f:
  File "/cm/shared/apps/python3/3.6.5sql/lib/python3.6/zipfile.py", line 1108, in __init__
    self._RealGetContents()
  File "/cm/shared/apps/python3/3.6.5sql/lib/python3.6/zipfile.py", line 1175, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
`

When I try to manually unzip the file, it also fails with such an error. Did I miss something or is there something wrong with the archive?

Thanks a lot

**Comment by weihua916** [comment]
Hi! This might be related to the storage issue. See https://github.com/snap-stanford/ogb/issues/79
